### ISSUE TITLE
A11y: password generator options should ideally have clearer accessible names/labels

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/desktop#1460]
### Steps To Reproduce

1. Using a screen reader, navigate to the password generator
2. Open the "Options"
3. Using `Tab`, set focus to the `A-Z`, `a-z`, `0-9`, `!@#$%^&*` checkboxes

### Expected Result

The checkboxes should be announced in an understandable/sensible way to screen reader users.

### Actual Result

`A-Z` and `a-z` are announced exactly the same way, `0-9` is announced as "zero nine", `!@#$%^&*` is announced as "at number dollar percent and star" (as screen readers will, by default, not announce some of those punctuation marks when presented like that as a nonsensical string in particular).

### Additional Context

Same issue as https://github.com/bitwarden/browser/issues/1985

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.32.1 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64

**Comment by patrickhlauke** [comment]
@Hinton suggest this is also something that might be good to backport to the website itself, for consistency
